ActindoWorkFlow/UserInterface/08_CoreActions.md

```diff
@@ -0,0 +1,247 @@
+
+  ![PHP code editor](../../Assets/Screenshots/ActindoWorkFlow/Workflows/CoreActions/PHPCodeEditor.png "[PHP code editor]")
+
+
+
+## Split by criterion
+
+![Split by criterion](../../Assets/Screenshots/ActindoWorkFlow/Workflows/CoreActions/SplitByCriterion.png "[Split by criterion]")
+
+The *Split by criterion* core action is used to compare the input value with a certain criterion and if it matches, a different branch is followed up than if it differs.   
+The data runs via the *in* input port into the workflow action and is either output via the *match* output port if the input value matches the criterion to be compared with, or via the *noMatch* output port if the input value does not match the criterion to be compared with.   
+The criterion to be compared with is defined in the configuration.
+
+> [Info] It is recommended to give the action a name that describes with which criterion the input value is compared.
+
+### Configuration  
+
+- *Path*   
+  Enter the path to the property of the input object to be compared with.
+
+  > [Info] The properties of an entity you can refer to are described in the relevant API documentation.
+
+
+- *Operator*   
+  Enter the applicable operator for the comparison. The following operators are available:
+  - **=**  
+    It is a match if the input value equals the defined value.
+  - **!=**    
+    It is a match if the input value does not equals the defined value.
+  - **in**    
+    It is a match if the input value is included in the defined value. For this operator, it is possible to enter an array in the *Value* field.
+  - **notIn**   
+    It is a match if the input value is not included in the defined value. For this operator, it is possible to enter an array in the *Value* field.
+  - **regexp**   
+    It is a match if the input value equals the regular expression entered in the *Value* field.    
+    For detailed information about regular expressions, see https://regex101.com/.
+
+
+- *Value*   
+  Enter the value to be compared with the input value. The value must be valid JSON and is type strict, for example a string must be specified as a string (**\"example\"**), an integer as an integer (**123**), and so on.
+
+[comment]: <> (Weitere operatoren geplant: <=, <, >=, >)
+
+
+
+## Start subprocess
+
+![Start subprocess](../../Assets/Screenshots/ActindoWorkFlow/Workflows/CoreActions/StartSubprocess.png "[Start subprocess]")
+
+The *Start subprocess* core action is used to start a different subprocess within the currently running process.       
+The data runs via the *p* input port into the workflow action and is output via the *p* output port. However, the action does not wait until the subprocess has been finished, but outputs the data after having started the subprocess. The input data is also the data put in the start place of the subprocess.   
+The subprocess to be started is defined in the configuration.
+
```

Comment:
Allgemeine Frage: Warum jeder Satz in einer neuen Zeile und nicht alle in einem Abschnitt?